pkg/schedule/schedulers/balance_leader.go

```diff
@@ -52,8 +51,7 @@ const (
 )
 
 type balanceLeaderSchedulerConfig struct {
-	syncutil.RWMutex
-	schedulerConfig
+	baseDefaultSchedulerConfig
```

Comment:
I suggest moveing `Disabled` here.

Reply:
If this, we should implement `isDisable` and `setDisable` four times.

Reply:
friendly ping~

Reply:
Just like other config?